### PR TITLE
Multiple responsible contributors

### LINKED ISSUE #896: Multiple responsible lecturers
In some cases multiple lecturers are equally responsible for a course. What would we need to reflect that on the platform?

### PR DESCRIPTION
**Pull request by jakobmache**
Fix #896

### CODE REVIEW

**Comment by karyon** [comment]
I had a look into the ordering thing, in the end the solution was trivial but i don't fully understand why it works but we have a test for that so i guess it's fine.

i searched for remaining places where `responsible_contributor` is accessed (using regex `responsible_contributor[^s]` and found two more, could you search and fix those as well? then this should be good to go.

**Comment by karyon** [comment]
oh and there's one remaining landscape issue. maybe just rename the index parameter to i.

File: evap/staff/forms.py [hunk 1]
```diff
@@ -471,8 +467,6 @@ def clean(self):
 
         if count_responsible < 1:
             raise forms.ValidationError(_('No responsible contributor found. Each course must have exactly one responsible contributor.'))
```
Comment (janno42) [inline evap/staff/forms.py]: This message must be updated

File: evap/grades/templates/grades_semester_view.html [hunk 2]
```diff
@@ -21,7 +21,7 @@ <h3 class="panel-title">{% trans "Courses" %}</h3>
                     <thead>
                         <tr>
                             <th class="col-sm-5">{% trans "Name" %}</th>
-                            <th class="col-sm-3">{% trans "Responsible" %}</th>
+                            <th class="col-sm-3">{% trans "Responsibles" %}</th>
```
Comment (janno42) [inline evap/grades/templates/grades_semester_view.html]: please keep `Responsible`. i will update the translation to `Verantwortlich`.

File: evap/results/templates/results_course_detail.html [hunk 3]
```diff
@@ -64,7 +64,7 @@
                     <tr>
                         <th class="col-xs-1">{% trans "Degree" %}</th>
                         <th class="col-xs-2">{% trans "Type" %}</th>
-                        <th class="col-xs-3">{% trans "Responsible" %}</th>
+                        <th class="col-xs-3">{% trans "Responsibles" %}</th>
```
Comment (janno42) [inline evap/results/templates/results_course_detail.html]: same here

File: evap/results/templates/results_semester_detail.html [hunk 4]
```diff
@@ -22,7 +22,7 @@
                         <tr>
                             <th class="col-sm-4">{% trans "Course" %}</th>
                             <th class="col-sm-1">{% trans "Type" %}</th>
-                            <th class="col-sm-3">{% trans "Responsible" %}</th>
+                            <th class="col-sm-3">{% trans "Responsibles" %}</th>
```
Comment (janno42) [inline evap/results/templates/results_semester_detail.html]: same here

File: evap/results/templates/results_semester_detail.html [hunk 5]
```diff
@@ -77,7 +81,7 @@
...
```
Comment (janno42) [inline evap/results/templates/results_semester_detail.html]: same here

**Pull request merged**